### ISSUE TITLE
Confused about loss in depth estimation example

### ISSUE THREAD

**Issue by aliasghar53** [keras-team/keras-io#969]
I'm following this example:
https://github.com/keras-team/keras-io/blob/master/examples/vision/depth_estimation.py

However, I'm confused as to why the model is compiled with the cross-entropy loss when a custom loss function is being used for optimization?

**Comment by pcoet** [comment]
Hi @aliasghar53, thanks for reaching out. The issue backlog is not the best place for support requests. Can you repost this to one of the forums below? That way your question will have more visibility.

* [TensorFlow forum](https://discuss.tensorflow.org/)
* [Keras mailing list](https://groups.google.com/forum/#!forum/keras-users)

Thanks!

**Comment by SuryanarayanaY** [comment]
Hi @aliasghar53 ,

Reported issue got fixed with the linked PR above. Thanks!
